src/index.js

```diff
@@ -213,7 +213,7 @@ function createLoadableComponent(loadFn, options) {
         });
     }
 
-    componentWillUnmount() {
+    componentDidUnmount() {
```

Comment:
As I understand, `componentWillUnmount` wasn't deprecated, only `componentWillMount`

Reply:
Okay , Sir I will change it

Reply:
Sir i have changed the method back to componentWillUnmount